gcp/cloud-endpoints/base/profile.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeflow.org/v1alpha1
+kind: Profile
+metadata:
+  name: kubeflow-admin
```

Comment:
Should this be kubeflow-${email}

Reply:
would not be a valid name 
```
$ kubectl create ns kubeflow-kam.d.kasravi@intel.com
The Namespace "kubeflow-kam.d.kasravi@intel.com" is invalid: metadata.name: Invalid value: "kubeflow-kam.d.kasravi@intel.com": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
Can we make it a param so `kubeflow-admin` is default value and kfctl code can overwrite?
We can convert email to valid name in kfctl.